Src/BootCamp.Chapter/CaesarCipher.cs

```diff
@@ -11,12 +10,46 @@ public static class CaesarCipher
     {
         public static string Encrypt(string message, byte shift)
         {
-            return message;
+            return string.IsNullOrEmpty(message) ? message : EncryptedMessage(message,shift); 
+        }
+
+        private static string EncryptedMessage(string message, byte shift)
+        {
+            byte[] asciiBytes = Encoding.ASCII.GetBytes(message);
+
+            for (int i = 0; i < asciiBytes.Length; i++)
+            {
+                int newAsciiNumber = (asciiBytes[i] + shift) % 255;
+                asciiBytes[i] = (byte)newAsciiNumber;
+            }
+
+            char[] asciiChars = Encoding.ASCII.GetChars(asciiBytes);
+            return new string(asciiChars);
         }
 
         public static string Decrypt(string message, byte shift)
         {
-            return message;
+            return string.IsNullOrEmpty(message) ? message : DecryptedMessage(message, shift);
+        }
+
+        private static string DecryptedMessage(string message, byte shift)
```

Comment:
Decrypt is the same as encrypt (except the shift is different). Please refactor

Reply:
Thanks for the feedback. I added a bool parameter to EncryptedMessage to check if it should encrypt or decrypt.